pkg/controllers/resources/persistentvolumeclaims/syncer_test.go

```diff
@@ -95,15 +95,13 @@ func TestSync(t *testing.T) {
 			Name:      pObjectMeta.Name,
 			Namespace: pObjectMeta.Namespace,
 			Annotations: map[string]string{
-				translate.NameAnnotation:               vObjectMeta.Name,
-				translate.NamespaceAnnotation:          vObjectMeta.Namespace,
-				translate.UIDAnnotation:                "",
-				translate.KindAnnotation:               corev1.SchemeGroupVersion.WithKind("PersistentVolumeClaim").String(),
-				translate.ManagedAnnotationsAnnotation: "otherAnnotationKey",
```

Comment:
why is this removed?

Reply:
The "otherAnnotationKey" and "translate.ManagedAnnotationsAnnotation" are removed because the object that was passed in doesn't have these annotations, so the syncer is correctly removing these. If we want to keep them we need to create a new object that has these as well, but I don't think this needs to be part of this testing logic. It worked before because the syncer was not updating metadata in this step and would require a second Sync call to remove that metadata

Reply:
makes sense, no need to keep it then I agree